src/core/column/column.service.js

```diff
@@ -1,10 +1,28 @@
+import {isFunction} from '../services/utility';
+import AppError from '../infrastructure/error';
+
 export function map(columns) {
 	return columns.reduce((memo, column) => {
 		memo[column.key] = column;
 		return memo;
 	}, {});
 }
 
+export function getValue(column) {
```

Comment:
as i wrote this shouldn't be here, you need to export it to pipe context as valueFactory from value.js